helm/templates/_helpers.tpl

```diff
@@ -5,6 +5,66 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Name of k8s cluster
+*/}}
+{{- define "kubernetes-cluster.name" -}}
+{{- if empty .Values.clusterName }}
+{{- default "" (printf "k8s-cluster-%s" (sha256sum .Release.Name | trunc 7)) }}
```

Comment:
Is the release name unique enough for our purposes here? Is a hash good enough or do we need randomness?
If the same release is installed on multiple clusters with the same name, wont all the clusters have the same hash?

Reply:
This is a good point, and we should add some randomness to it.